### ISSUE TITLE
Is it possible to share more code?

### ISSUE THREAD

**Issue by wzm2256** [kanezaki/pytorch-unsupervised-segmentation-tip#1]
Hi, I have some questions concerning the evaluation and the dataset. 

1. Is it possible to share the code to produce the mIOU score?
Since this IOU is used when the number of clusters is different between GT and outputs,  which is fundamentally different from mIOU in other applications. It would be nice if you can share the code so followers can use it without misunderstanding.

2. In Table 1, did you use all data in VOC, BSD datasets? or only the training set? or the validation set? or just a few of them displayed in your paper?

3. In Table 1, if my understanding is correct, the algorithm is performed on ALL images in these datasets one by one. How long does that take? Since one run takes several minutes, I suppose that would be a rather LONG time.

4. Is it possible to share the code to pre-process the VOC dataset?  As you mentioned in your paper "the object category labels in PASCAL VOC 2012 dataset were ignored and each segment along with the background region was treated as an individual segment." I'm not sure what that means.

Thanks for your time. It would be nice if you can answer these questions.
